src/target/riscv/riscv.h

```diff
@@ -326,17 +326,33 @@ struct riscv_info {
 
 	bool range_trigger_fallback_encountered;
 
-	bool riscv_ebreakm;
-	bool riscv_ebreaks;
-	bool riscv_ebreaku;
-
 	bool wp_allow_equality_match_trigger;
 	bool wp_allow_napot_trigger;
 	bool wp_allow_ge_lt_trigger;
 
 	bool autofence;
 };
 
+enum riscv_ebreak_ctl {
+	RISCV_DCSR_EBREAK_M_CTL,
+	RISCV_DCSR_EBREAK_S_CTL,
+	RISCV_DCSR_EBREAK_U_CTL,
+	RISCV_DCSR_EBREAK_VS_CTL,
+	RISCV_DCSR_EBREAK_VU_CTL,
+	N_RISCV_DCSR_EBREAK_CTL
+};
```

Comment:
This enum is in fact a list of RISC-V privilege modes, so I would recommend to rename it in that fashion.

Once renamed, it will also become possible to reuse the enum elsewhere else.

Example:

```suggestion
enum riscv_priv_mode {
	RISCV_MODE_M,
	RISCV_MODE_S,
	RISCV_MODE_U,
	RISCV_MODE_VS,
	RISCV_MODE_VU,
	N_RISCV_MODE
};
```

Reply:
Addressed.